frame/contracts/src/tests.rs

```diff
@@ -372,39 +372,10 @@ fn account_removal_removes_storage() {
 	});
 }
 
-const CODE_RETURN_FROM_START_FN: &str = r#"
-(module
-	(import "env" "ext_return" (func $ext_return (param i32 i32)))
-	(import "env" "ext_deposit_event" (func $ext_deposit_event (param i32 i32 i32 i32)))
-	(import "env" "memory" (memory 1 1))
-
-	(start $start)
-	(func $start
-		(call $ext_deposit_event
-			(i32.const 0) ;; The topics buffer
-			(i32.const 0) ;; The topics buffer's length
-			(i32.const 8) ;; The data buffer
-			(i32.const 4) ;; The data buffer's length
-		)
-		(call $ext_return
-			(i32.const 8)
-			(i32.const 4)
-		)
-		(unreachable)
-	)
-
-	(func (export "call")
-		(unreachable)
-	)
-	(func (export "deploy"))
-
-	(data (i32.const 8) "\01\02\03\04")
-)
-"#;
-
 #[test]
 fn instantiate_and_call_and_deposit_event() {
-	let (wasm, code_hash) = compile_module::<Test>(CODE_RETURN_FROM_START_FN).unwrap();
+	let (wasm, code_hash) = compile_module::<Test>(include_str!("tests/return_from_start_fn.wat"))
```

Comment:
This doesn't solve the issue described:
> Changing the contents of a .wat file causes recompilation of the whole crate.

Reply:
The fix is to use https://doc.rust-lang.org/std/fs/fn.read_to_string.html.